game/plugins/src/main/kotlin/gg/rsmod/plugins/content/skills/thieving/pickpocketing/pickpocket.kts

```diff
@@ -0,0 +1,69 @@
+package gg.rsmod.plugins.content.skills.thieving.pickpocketing
+
+import gg.rsmod.plugins.content.combat.isAttacking
+import gg.rsmod.plugins.content.combat.isBeingAttacked
+
+//for each npc in the enum
+PickpocketInfo.values().forEach { npcClass ->
+
+    //for each npc id in the current enum
+    npcClass.ids.forEach { npcID ->
+
+        //Extra check to ensure that the npc is pickpocket-able
+        if (world.definitions.get(NpcDef::class.java, npcID).options.contains("Pickpocket")) {
```

Comment:
Rather seeing this like `if(!world.defi......) {continue;}` 
the foreach statements and if statements are already stacking up quite a bit so this would make it a bit cleaner.